Homework10/DerivedClass.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework10
+{
+    internal class DerivedClass : FileWorker
+    {
+        string maxSizeType = "txt";
```

Comment:
maxSizeType "txt"  size არ არის ფაილის გაფართოებაა

Reply:
eg rogor unda gamometana? file is gafartoeba?

Reply:
Cvaladis saxelia ubralod araswori 
Files gafartoeba rogor gamoatanino magas orshabats gavivlit

Reply:
aha okay